data/src/error_analysis/erroranalysis.py

```diff
@@ -0,0 +1,70 @@
+"""Error Analysis tool for G2P.
+
+This script assumes 2 input files. a) Covering Grammer b) Test output. 
+CG file  : Each line contain graphime and their corresponding pronunciation seperated by a tab
+Test File: contains three attributes in each line seperated by a tab. The attributes are - orthography, expected pronunciation, and hypothesized pronunciation."""
+
+__author__ = "Arundhati Sengupta"
+
+import pynini
+import argparse
+
+def main(args: argparse.Namespace):
+    cg_fst = pynini.string_file(args.cg_path).closure().optimize()
+    rulematch_predMatch = 0
```

Comment:
these are Java-style camelCase variables whereas in Python we use snake_case for identifiers.

Reply:
fixed.